apollo-compiler/src/test/graphql/com/example/hero_details/HeroDetails.kt

```diff
@@ -213,13 +222,14 @@ class HeroDetails : Query<HeroDetails.Data, HeroDetails.Data, Operation.Variable
 
   companion object {
     const val OPERATION_ID: String =
-        "257332d822c9bcd5dabeff3f3dda46875a47846f6eeae88f9042c94e3effeee7"
+        "e9e881883e577da3a4dc0ea9eedbdbc8a05f65fe08bd6f1ae6c1e993b75dfbe4"
 
     val QUERY_DOCUMENT: String = QueryDocumentMinifier.minify(
           """
           |query HeroDetails {
           |  hero {
           |    __typename
+          |    type
```

Comment:
This looks like an issue. It shouldn't have extra spaces. Kotlin's trim function may get confused here.

Reply:
Nevermind, it looks like I shouldn't trust GitHub mobile app for code reviews. The indentation was completely off there.